src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper implements DtoResponseMapper<ShoppingCartResponseDto,
+        ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto shoppingCartResponseDto = new ShoppingCartResponseDto();
+        shoppingCartResponseDto.setId(shoppingCart.getId());
+        shoppingCartResponseDto.setTicketIds(shoppingCart.getTickets());
```

Comment:
Instead of setting the whole Ticket objects, you should set a list of ticket IDs.